src/tests/linearElementEditor.test.tsx

```diff
@@ -25,26 +25,29 @@ import {
 import * as textElementUtils from "../element/textElement";
 import { ROUNDNESS, VERTICAL_ALIGN } from "../constants";
 
-const renderScene = jest.spyOn(Renderer, "renderScene");
+// FIXME I: add specific tests for render of both components (when they should both, when just one, when just second, what is the order of renders - first static, then interactive, etc., all tests)
```

Comment:
Still want to re-check these component tests for the rendering flow & react.memo controller.